website/docs/contributing/reference/verify_a_release_candidate.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Verify a release candidate
+---
+
+To verify a release candidate, the following checklist could be used:
+
+- [ ] Download links are valid.
+- [ ] Checksums and signatures.
+- [ ] LICENSE/NOTICE files exist.
+- [ ] No unexpected binary files.
+- [ ] All source files have ASF headers.
+- [ ] Can compile from source.
+
+:::note
+
+It is NOT necessary to run all checks to cast a vote for a release candidate.
+
+However, you should clearly state which checks you did. The release manager needs to ensure that each check was done.
+
+:::
+
+## Download links are valid
+
+To verify the release candidate, you need to download the release candidate from the [dist](https://dist.apache.org/repos/dist/dev/incubator/opendal/) directory.
+
+Use the following command to download all artifacts, replace "${release_version}-${rc_version}" with the version ID of the version to be released:
+
+```shell
+svn co https://dist.apache.org/repos/dist/dev/opendal/${release_version}-${rc_version}/
+```
+
+## Checksums and signatures
+
+The release candidate should have a checksum and signature file. 
+
+For example, if the release candidate is `0.36.0-rc1`, the checksum and signature file should be:
+
+```
+https://dist.apache.org/repos/dist/dev/incubator/opendal/0.36.0-rc1/apache-opendal-0.36.0-rc1-src.tar.gz.sha512
+https://dist.apache.org/repos/dist/dev/incubator/opendal/0.36.0-rc1/apache-opendal-0.36.0-rc1-src.tar.gz.asc
+```
+
+### Verify checksums and signatures
+
+GnuPG is recommended here. It can be installed with the following command:
+
+```shell
+apt-get install gnupg
+# or
+yum install gnupg
+# or
+brew install gnupg
+```
+
+Firstly, import the OpenDAL release manager's public key:
+
+```shell
+curl https://downloads.apache.org/opendal/KEYS > KEYS # Download KEYS
+gpg --import KEYS # Import KEYS to local
+```
+
+Then, trust the public key:
+
+```shell
+gpg --edit-key <KEY-used-in-this-version> # Edit the key
+```
+
+It will enter the interactive mode, use the following command to trust the key:
+
+```shell
+gpg> trust
+```
+
+And then, select the level of trust, for example:
+
+```
+Please decide how far you trust this user to correctly verify other users' keys
+(by looking at passports, checking fingerprints from different sources, etc.)
+
+  1 = I don't know or won't say
+  2 = I do NOT trust
+  3 = I trust marginally
+  4 = I trust fully
+  5 = I trust ultimately
+  m = back to the main menu
+```
+
+Select `5` to trust the key ultimately.
+
+Now, we could start the verification. 
+
+Save the following content as `verify.sh`:
```

Comment:
Why not using our existing verify script?

Reply:
This content is copied from our script. 
It is listed separately here so people who want to verify don't need to check out the whole repo.

Reply:
> It is listed separately here so people who want to verify don't need to check out the whole repo.

I prefer them to checkout our repo instead so that we don't need to make sure the script list here is consist with our repo.

Reply:
Updated.